### ISSUE TITLE
The remaining controls in the find/replace window can't be tabbed to

### ISSUE THREAD

**Issue by cannona** [microsoft/vscode#280340]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.77.0 is when it seems to have first appeared, commit 58e3dd3f033a appears to be the cause, and it happens up until the latest build.
- OS Version: Windows 10, 11, and Debian Trixie

Steps to Reproduce:

1. Open an existing or new file.
2. press CTRL+f to access the find/replace window.
3. hit the tab key and notice that the focus doesn't move.
4. Press shift+tab to move to the find/replace toggle, and turn it on.
5. Tab forward twice to land back in the find box, and then the replace box.
6. Press Tab again, and notice that the focus once again does not move to the rest of the controls in the window.
7. Press CTRL+m to change the tab key mode, and notice that it still doesn't work.

I am using a screen reader, if that matters.

**Comment by cannona** [comment]
> hi [@cannona](https://github.com/cannona) , may I ask how you found out it was commit [58e3dd3](https://github.com/microsoft/vscode/commit/58e3dd3f033a53972103053a95182f887d11f37f) that caused the issue? did you use vscode-bisect or git bisect or maybe a different technique altogether? :)

I didn't know `vscode bisect` was a thing. I used `git bisect`.
